### PR TITLE
[NOREF] - Organize SQL Queries

### PR DESCRIPTION
**Pull request by rileyanderson**
## Changes and Description

- Move SQL queries to folders
- structured in `SQL/{table_name}/{query}.sql` in a few cases it is structured from the name of the `*_Store.go` file where it makes more sense to do so (e.g. `SQL/access_controll/**`)

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

File: pkg/storage/SQL/check_if_collaborator_by_operational_need_id.sql [hunk 1]
```diff
@@ -1,7 +0,0 @@
-SELECT EXISTS(
```
Comment (StevenWadeOddball) [inline pkg/storage/SQL/check_if_collaborator_by_operational_need_id.sql:1]: Why was this deleted?

Comment (rileyanderson) [reply pkg/storage/SQL/check_if_collaborator_by_operational_need_id.sql:1]: It wasn't used anywhere

Comment (StevenWadeOddball) [reply pkg/storage/SQL/check_if_collaborator_by_operational_need_id.sql:1]: I think it can be left for now, we are going to need it fairly soon (for operational solution access control).

Comment (rileyanderson) [reply pkg/storage/SQL/check_if_collaborator_by_operational_need_id.sql]: Added it back

**Review by ClayBenson94 — APPROVED** [review]
This looks good! Thanks for running through all these - should help when we introduce more .sql files as well 😄
